runtime/local/src/lib.rs

```diff
@@ -829,6 +835,24 @@ pub type Executive = frame_executive::Executive<
     AllPalletsWithSystem,
 >;
 
+#[derive(Default)]
+pub struct DappsStakingChainExtension;
+
+impl RegisteredChainExtension<Runtime> for DappsStakingChainExtension {
+    const ID: u16 = 00;
+}
+
+impl ChainExtension<Runtime> for DappsStakingChainExtension {
+    fn call<E: Ext>(&mut self, env: Environment<E, InitState>) -> Result<RetVal, DispatchError>
+    where
+        E: Ext<T = Runtime>,
+        <E::T as SysConfig>::AccountId: UncheckedFrom<<E::T as SysConfig>::Hash> + AsRef<[u8]>,
+    {
+        DappsStakingExtension::execute_func::<E>(env.func_id().into(), env)?;
```

Comment:
I think here you only need to return 
`DappsStakingExtension::execute_func::<E>(env.func_id().into(), env)` as it return a RetVal